packages/fxa-auth-server/lib/senders/emails/scss/style.scss

```diff
@@ -0,0 +1,30 @@
+$large-font-size: 20px;
+$small-font-size: 12px;
```

Comment:
-dances- I know I'm the one that suggested these names for our variables, but looking at the [Tailwind class names for font-size](https://tailwindcss.com/docs/font-size) and noting those classes also set `line-height`, I wonder if we could just copy the Tailwind naming conventions and their associated styles (using `px` instead of `rem` since it's safer for emails, meaning you'll multiply those rem values by `16`). For `text`, we could use [`@mixin`](https://sass-lang.com/documentation/at-rules/mixin)s.

It'd look something like this:

```
@mixin text-xs {
  font-size: 12px !important;
  line-height: 16px !important;
}
```

And then when you need to use it:
```
.primary-text div {
  @include text-small;
  font-family: sans-serif !important;
  font-weight: normal !important;
}
```

We could potentially use a Sass [placeholder](https://sass-lang.com/documentation/style-rules/placeholder-selectors) instead (`%text-sm` syntax), though mixins give us the option to add arguments later, like if we _really_ want a custom line-height for some reason so I feel it's more appropriate here. 

We can also add other patterns we identify to something like this - maybe `font-family: sans-serif` + `font-weight: normal` comes up enough that we want a custom mixin for that? Not sure, but we can probably set some `margin`, `padding`, and `color`s with regular SCSS variables.

Checking the [Tailwind padding docs](https://tailwindcss.com/docs/padding), `4px` would be `p-1`. So we could set:

```
$p-1: 4px !important;
```
Then:
```
td {
  padding: $p-1;
}
```

I don't think we need to go overboard, but I'll be super happy with how much we can DRY it up if we can force some consistency between `font-size`, `line-height`, `padding`, `margin`, and color properties. 👍 

I'm curious to hear @jodyheavener's thoughts on this.

Reply:
Yes I agree with this. Our emails have a fairly limited set of styles that often come grouped together, so this is a good approach IMO. My only hesitation would be if we eventually start more liberally grouping common styles together, and then eventually the mixins get just as bogged down as element-specific rulesets. But I'm less concerned with email styles.